src/lib/libast/path/pathbin.c

```diff
@@ -19,28 +19,30 @@
  *                     Phong Vo <phongvo@gmail.com>                     *
  *                                                                      *
  ***********************************************************************/
-/*
- * Glenn Fowler
- * AT&T Bell Laboratories
- *
- * return current PATH
- */
+//
+//  Glenn Fowler
+//  AT&T Bell Laboratories
+//
 #include "config_ast.h"  // IWYU pragma: keep
 
 #include <string.h>
 
 #include "ast.h"
 
+//  Return current PATH
+//
 char *pathbin(void) {
-    char *bin;
+    char *path;
+    // Used to restore value of saved astconf path from earlier function call
+    static char *astconf_path;
```

Comment:
I know it's implicitly initialized to zero but it's better to add an explicit `= NULL;` initializer.